### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by halkeye** [halkeye/zabox#10]
Readme file with useful information Ideas? 
- [ ] Description
- [ ] screenshots

What else should be done before next release?

**Comment by halkeye** [comment]
Installation Instructions

**Comment by halkeye** [comment]
Contributing?

**Comment by halkeye** [comment]
What should be next version? 1.2?

**Comment by severest** [comment]
Permanent demo?

**Comment by halkeye** [comment]
> Permanent demo?

I was thinking about that, heroku would be good for this, except it I don't think smtp server could be exposed 'cause it needs a second port. I don't really want to open up the second port on my server either.

Is the test storage engine good enough?

**Comment by severest** [comment]
Yea, don't think a working SMTP port is necessary, just the demo messages may need some spicing up

**Comment by halkeye** [comment]
Agreed. I'll setup a heroku project. Do you have an account? I'll swap to
Travis to do deploys as well (always wanted an excuse to figure that out)

**Comment by halkeye** [comment]
Apparently heroku now has a "deploy on push to master" option, so it was a breeze.
https://zabox.herokuapp.com/#/messages
zabox.gavinmogan.com will point there when dns updates

**Comment by halkeye** [comment]
@severest i'm going to write up a draft of the readme soon with some screenshots. Any idea how the test emails should be modified?

**Comment by severest** [comment]
I'll take care of the test emails
